content/minfraud/alerts.mdx

```diff
@@ -71,6 +71,7 @@ Alert. These codes are subject to change.
 | `HIGH_RISK_IP`     | IP address has been marked as a high-risk IP                                                     |
 | `HOSTING_PROVIDER` | IP is from High Risk Hosting Provider                                                            |
 | `POSTAL_VELOCITY`  | IP address had high velocity of orders (e.g. different zipcodes on same IP address)              |
+| `MANUAL_REVIEW`    | The transaction was flagged as risky after manual review                                         |
```

Comment:
A small nit: the other reasons are intentionally in alphabetic order.

Reply:
Ah sorry I totally missed that, i have a few more PRs coming out tomorrow so I can also move this in the correct order then. Thank you for flagging!

Reply:
Oops, my bad. Apologies for the inconvenience.